app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= simple_form_for(@review) do |f| %>
```

Comment:
Potentially needs to be [@booking, @review] as the booking ID will need to be defined when a review is made, but this will need testing for sure.